src/dao/pages/DAOsListPage/DAOsListPage.tsx

```diff
@@ -28,22 +54,67 @@ export function DAOsListPage() {
         />
       }
       body={
-        <CardGridLayout<DAO[]>
-          columns={{ md: 2, lg: 3 }}
-          queryResult={daos}
-          message={"It looks like no DAOs have been created yet."}
-          preLinkText={"Click on this link to&nbsp;"}
-          linkText={"create the first DAO"}
-          onLinkClick={handleLinkClick}
-        >
-          {daos.data?.map((dao: DAO, index: number) => {
-            const { accountEVMAddress, name, type, isPrivate, logoUrl } = dao;
-            if (isPrivate) return null;
-            return (
-              <DAOCard key={index} accountEVMAddress={accountEVMAddress} name={name} type={type} logoUrl={logoUrl} />
-            );
-          })}
-        </CardGridLayout>
+        <Flex direction="column" gap={4}>
+          <Tabs defaultIndex={0} onChange={handleTabChange} bg={Color.White} variant="dao-dashboard-tab">
+            <TabList borderBottom="0">
+              <Tab>
+                <Box>All</Box>
+              </Tab>
+              <Tab>
+                <Box>My DAOs</Box>
+              </Tab>
+            </TabList>
+          </Tabs>
+          <Flex justifyContent="flex-end" gap={4}>
+            <Input
+              variant="filter"
+              value={searchText}
+              onChange={(e: any) => setSearchText(e.target.value)}
```

Comment:
`onChange` events can be typed as `ChangeEvent<HTMLSelectElement>`, e.g. `(event: ChangeEvent<HTMLSelectElement>)`

Reply:
Done.